src/App.tsx

```diff
@@ -14,11 +16,111 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type Props = {};
+
+type State = {
+  selectedGood: string[];
+};
+
+class App extends React.Component<Props, State> {
```

Comment:
also you could write like this:
```suggestion
type State = {
  selectedGood: string[];
};

class App extends React.Component<{}, State> {
```

Reply:
Yes, but I decided that in my way it would be easier to change props, if necessary.